lib/libraries/nunjucks.js

```diff
@@ -8,6 +8,7 @@ module.exports = (() => {
         'app/_layouts',
         'node_modules/govuk-frontend'
       ], {
+        noCache: true,
```

Comment:
Do we want to only apply `noCache` when `process.env.NODE_ENV === 'development'`?

Reply:
Possibly, although I'm not sure there is much difference when it comes to static server-side generation, I think that functionality is about optimising for client-side rendering. As Nunjucks is technically 'isomorphic' template engine.